### PR TITLE
Fix materialized view issues

### PR DESCRIPTION
**Pull request by derekprior**
We were lacking a smoke test for materialized view functionality. Adding it
revealed a couple of issues that needed to be fixed:

1. We never implemented `refresh_materialized_view`.

2. We could not successfully run the `rails destroy scenic:view` with the
   materialized option.

### CODE REVIEW

**Comment by calebthompson** [comment]
HAHAHA

**Comment by calebthompson** [comment]
:(

File: spec/smoke [hunk 1]
```diff
@@ -3,87 +3,113 @@
...
+
+  echo "rake db:rollback"
+  rake db:rollback
+  echo "[success]"
+
+  echo "rails destroy scenic:view search --materialized"
+  rails destroy scenic:model search --materialized
+  [[ ! -f app/models/search.rb ]] || exit 1
+  [[ ! -f db/views/searches_v01.sql ]] || exit 1
+  [[ ! -f app/models/search.rb ]] || exit 1
```
Comment (calebthompson) [inline spec/smoke]: This line duplicates 106

**Comment by calebthompson** [comment]
LGTM, though I'd prefer we didn't switch away from tabs.

**Comment by derekprior** [comment]
Bash style guides I can find are either indifferent on the matter or use spaces. Why do you want to use tabs?

**Comment by calebthompson** [comment]
Most of my shell style opinions are based on Git's CodingGuidelines.

https://github.com/git/git/blob/master/Documentation/CodingGuidelines#L43

**Comment by derekprior** [comment]
I'd like to move that to ruby in any event. I think it will enable more people to contribute and I'll certainly be more comfortable with it, anyway.

**Pull request merged**